cookbooks/cdo-apps/recipes/default.rb

```diff
@@ -23,6 +23,18 @@
   fonts-noto
 )
 
+replace_or_add 'Enable PDF in ImageMagick policy.xml' do
```

Comment:
Is there a reliable cookbook we could use to edit an XML file? Regex could be brittle. Or could we render the entire `policy.xml` with ERB?

Reply:
The policy file is pretty big, and I get nervous hardcoding that much boilerplate when we really just want to change one setting. Some kind of XML editing makes sense, or perhaps ensuring that this fails if the text to replace isn't found.